src/main.rs

```diff
@@ -15,17 +15,34 @@
 use jujutsu::cli_util::{create_ui, handle_command_result, parse_args, CommandError};
 use jujutsu::commands::{default_app, run_command};
 use jujutsu::ui::Ui;
+use tracing::metadata::LevelFilter;
+use tracing_subscriber::reload::Handle;
+use tracing_subscriber::{prelude::*, Registry};
 
-fn run(ui: &mut Ui) -> Result<(), CommandError> {
+fn run(ui: &mut Ui, reload_log_filter: Handle<LevelFilter, Registry>) -> Result<(), CommandError> {
     let app = default_app();
     let (command_helper, matches) = parse_args(ui, app, std::env::args_os())?;
+    if command_helper.global_args().verbose {
```

Comment:
There are two other examples of how to customize the `jj` binary in `examples/`. I suppose we'll want `-v` to work for those without duplicating this code, so some restructuring is needed. The call to `jujutsu::cleanup_guard::init()` would also make sense to move into a shared place. I'm not sure how to structure it. Maybe some `CliRunner` struct that could keep the `(filter, reload_log_filter)` pair? Feel free to just leave a TODO about if you prefer to do it after this PR has landed.

Reply:
I've added a TODO. I'll need to dig a bit more into how the cleanup guard and the Ui code works before I can make a reasonable change there I think.